lhotse/dataset/speech_synthesis.py

```diff
@@ -0,0 +1,73 @@
+import re
+from pathlib import Path
+from typing import Dict, List, Optional
+
+import torch
+from torch.utils.data import Dataset
+
+from lhotse.cut import CutSet
+from lhotse.utils import Pathlike
+
+EPS = 1e-8
+
+
+def text_to_tokens(text: str) -> List[str]:
+    text = re.sub(r'[^\w !?]', '', text)
+    text = re.sub(r'^\s+', '', text)
+    text = re.sub(r'\s+$', '', text)
+    text = re.sub(' +', ' ', text)
+    return list(text.upper())
+
+
+class SpeechSynthesisDataset(Dataset):
```

Comment:
Can you add a unit test for this dataset?

Reply:
I forgot to write unit tests. I'll add them soon.